doc/lua_api.txt

```diff
@@ -2842,8 +2842,10 @@ and `minetest.auth_reload` call the authentication handler.
         * decoration
     * The second parameter is a list of IDS of decorations which notification
       is requested for.
-* `get_gen_notify()`
+* `minetest.get_gen_notify()`
     * Returns a flagstring and a table with the `deco_id`s.
+* `minetest.get_deco_id(decoration_name)
+    * Returns the decoration ID for the provided decoration name string.
```

Comment:
\+ or \`nil\` on failure or when the decoration was not found.

Reply:
Yes good points, will change.

Reply:
Both done and retested.